test_src/cross_field_diffusion_tests.cpp

```diff
@@ -155,23 +188,33 @@ TEST_CASE( "cross-field diffusion operator - not fully implemented" )
       particleArray, dt, &state1.front(), perpDiffusionCoeff, nR_Bfield,
       nZ_Bfield, bfieldGridr.data(), &bfieldGridz.front(), &br.front(),
       &bz.front(), &by.front());
+    
+  // half-side length
+    double s = 0.2;
+    std::vector<gitr_precision> gold(net_nX-1,0.0);
+    for( int x_bin = 0; x_bin < net_nX; ++x_bin )
+    {
+      std::cout << "xbin " << x_bin << std::endl;
+      gold[x_bin] = 0.5/perpDiffusionCoeff*(s-gridX_midpoints[x_bin]);
```

Comment:
Are the experimental values aligning with these analytically calculated ones?  What is the rationale for these gold values?